src/swsssdk/configdb.py

```diff
@@ -142,18 +142,21 @@ def __raw_to_typed(self, raw_data):
         return typed_data
 
     def __typed_to_raw(self, typed_data):
+        raw_data = dict(); dfields = dict()
```

Comment:
if dfields is simply a collection of attributes with no value, it could be a list or set.